internal/server/handlers.go

```diff
@@ -332,6 +333,8 @@ func (h *handler) EndNotificationBatch() {
 func (h *handler) handleDeletionsFromIRV() {
 	for name, irv := range h.initialResourceVersions {
 		if _, ok := h.entries[name]; !ok && !irv.received {
+			slog.Debug("Resource no longer exists on the server but is still present on the client."+
```

Comment:
Nit: space is wrong here, the log line will be `"Resource no longer exists on the server but is still present on the client.Explicitly marking the resource for deletion"`.
```suggestion
			slog.Debug("Resource no longer exists on the server but is still present on the client. "+
```